integration/spark/app/src/test/java/io/openlineage/spark/agent/ConfigurableTestConfig.java

```diff
@@ -29,17 +31,22 @@
 import lombok.Setter;
 import lombok.SneakyThrows;
 import lombok.extern.slf4j.Slf4j;
+import org.junit.platform.commons.util.StringUtils;
 import org.testcontainers.shaded.org.yaml.snakeyaml.Yaml;
 import org.testcontainers.shaded.org.yaml.snakeyaml.constructor.Constructor;
+import org.testcontainers.utility.DockerImageName;
 
 @NoArgsConstructor
 @Getter
 @Setter
 @Slf4j
 public class ConfigurableTestConfig {
+
+  public static final Path IVY_DIR = Paths.get("/usr/lib/openlineage/integration/spark/cli/.ivy2");
```

Comment:
Why do we need that?

Reply:
So, `.ivy2` points to a location where packages added with `--packages` are stored. We wan't to have a Docker volume for this so that running multiple tests will lead do downloading package only once. 

However, when working with different Docker images, this is causing problems. A first container writes some xml files to `.ivy/cache` dir. Then a second container tries to read all the files from that directory and fails bcz of permission exception (this is happening on linux only where docker constrains are more strict). The solution to this is to clear `.ivy` cache directory `@BeforeAll`. 

The line you refer is just a constant variable used three times.